src/components/ui/ContextMenu/ContextMenu.tsx

```diff
@@ -0,0 +1,91 @@
+import { memo, useCallback } from 'react';
+import type { ReactNode } from 'react';
+import { useToggle } from 'react-use';
+
+import Tooltip from '@mui/material/Tooltip';
+import ClickAwayListener from '@mui/material/ClickAwayListener';
+
+import MoreIcon from '../../../assets/more.svg';
+import CrossIcon from '../../../assets/cross.svg';
+
+import Pressable from '../Pressable';
+import Typography from '../Typography';
+import HorizontalDivider from '../HorizontalDivider';
+
+import styles from './ContextMenu.module.scss';
+
+type ContextMenuOption = {
+  icon?: ReactNode;
+  label?: string;
+  onClick?: () => void;
+};
+
+type ContextMenuProps = {
+  options: Array<ContextMenuOption>;
+  trigger?: ReactNode;
+  testId?: string | null;
+};
+
+//  <MoreIcon />
+const ContextMenu = ({ options, trigger, testId = null }: ContextMenuProps) => {
```

Comment:
trigger should be `<MoreIcon />` by default

Reply:
I am setting that as default in line 80. If you really want me to set it as default here, we have to capitalize trigger

https://stackoverflow.com/questions/71018466/how-to-define-a-component-as-default-value-for-a-prop-in-react/71018639#71018639

Reply:
That's fine
The problem is that I don't see this default icon in storybook if we're not passing trigger

Reply:
So do you mean that you are fine with me not setting the default icon there, as long as I have fixed it?

Reply:
Yes. the main think is to see MoreIcon as default in storybook

Reply:
I think you have missed the trigger there. The trigger is located at the right on your screen.